packages/core-data/src/actions.js

```diff
@@ -644,12 +644,19 @@ export const saveEditedEntityRecord = (
 	if ( ! select.hasEditsForEntityRecord( kind, name, recordId ) ) {
 		return;
 	}
+	const entities = await dispatch( getKindEntities( kind ) );
```

Comment:
something feels weird here. should this be `select` or  `dispatch`?

Reply:
I just copied this over from `saveEntityRecord`. `getKindEntities` is an action, probably because it does `entities = await kindConfig.loadEntities();`. I wonder if it could be a selector with a resolver instead? But that's for another PR perhaps.

Reply:
https://github.com/WordPress/gutenberg/blob/88d29569067ddbd7eb7a47fed9bf70b4aa592b46/packages/core-data/src/actions.js#L383-L395

Reply:
Indeed I guess my old self is to blame for this confusion :P

Reply:
This code should actually be like:
```js
const entities = await resolveSelect.getEntitiesByKind( kind );
```
The `getEntitiesByKind` selector already exists, but it doesn't have a resolver attached. So we need to manually dispatch the `getKindEntities` action that fetches and populates the state read by `getEntitiesByKind`.